app/app.js

```diff
@@ -10,20 +9,25 @@ function Chicken (chix)  {
   this.publishedOn = chix.publishedOn;
   this.description = chix.description;
   this.image = chix.image;
+  this.toHtml = function() {
+    this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+    var template = $('#chickenTemplate').html();
+    var compiledTemplate = Handlebars.compile(template);
+    // $('#chickens').append(compiledTemplate(this));
+    return compiledTemplate(this);
+  };
 }
 
-Chicken.prototype.toHtml = function() {
-  this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
-  var template = $('#chickenTemplate').html();
-  var compiledTemplate = Handlebars.compile(template);
-  $('#chickens').append(compiledTemplate(this));
-};
-
 //passes constructed objects to allChickens
 rawChickenData.forEach(function(ele){
   allChickens.push(new Chicken(ele));
 });
 
+//sorts allChickens title alphabeticaly
+allChickens.sort(function(a,b){
+  return (a.title > b.title);
+});
+
```

Comment:
Script below to populate filters is really a View process and should be in that javascript file.